certbot/display/ops.py

```diff
@@ -289,3 +289,49 @@ def _gen_https_names(domains):
             domains[-1])
 
     return ""
+
+
+def _get_validated(method, validator, message, default=None, **kwargs):
+    if default is not None:
+        try:
+            validator(default)
+        except errors.PluginError as error:
+            raise AssertionError("Invalid default {0}".format(default))
```

Comment:
It seems like `while True` is an established pattern in the codebase, whereas I can't find uses of `for _ in range`. I can see the advantages of the `range` pattern, but I think it'd be better to make a change like that in a cross-cutting way if it's what we want.